wrangler-transform/src/main/java/io/cdap/wrangler/Wrangler.java

```diff
@@ -385,9 +387,11 @@ public void transform(StructuredRecord input, Emitter<StructuredRecord> emitter)
           emitter.emitError(new InvalidEntry<>(error.getCode(), error.getMessage(), input));
           errorMessages.add(error.getMessage());
         }
-        if (onErrorStrategy.equalsIgnoreCase("fail-pipeline")) {
-          throw new Exception(
-            String.format("Errors in Wrangler Transformation - %s", errorMessages));
+        if (WRANGLER_FAIL_PIPELINE_FOR_ERROR.isEnabled(getContext())) {
+          if (onErrorStrategy.equalsIgnoreCase("fail-pipeline")) {
```

Comment:
nit: define a const for `"fail-pipeline"`

Reply:
Updated.